python/ray/serve/_private/autoscaling_state.py

```diff
@@ -311,19 +318,22 @@ def _collect_handle_running_requests(self) -> List[List[TimeStampedValue]]:
 
         return timeseries_list
 
-    def _aggregate_ongoing_requests(
-        self, metrics_timeseries_dicts: List[Dict[str, List[TimeStampedValue]]]
+    def _aggregate_timeseries_metric(
+        self,
+        metrics_timeseries_dicts: List[Dict[str, List[TimeStampedValue]]],
```

Comment:
would recommend just passing in the list directly and check before invoking the call if metrics key is present. Also would be nice if we had a `TimeSeries` type, we could alias it to `List[TimeStampedValue]`.

```suggestion
        timeseries_list: List[TimeSeries],
```